forge-game/src/main/java/forge/game/ability/effects/ChangeZoneEffect.java

```diff
@@ -603,6 +603,17 @@ private void changeKnownOriginResolve(final SpellAbility sa) {
                             // If it can't Transform, don't change zones.
                             continue;
                         }
+                    } else if (sa.hasParam("FrontFace")){
```

Comment:
hmn was that really needed?

if you exile it the engine should already make sure it doesn't stack on its backside

Reply:
Yeah, I just didn't test things properly before adding that param